### DISCUSSION TITLE
How to change default field table from executing: php artisan make:filament-user

### DISCUSSION THREAD

**Discussion by siabapet** [filamentphp/filament#9123] [Help] [answered]
I got an error when i execute `php artisan make:filament-user` after i had changing the default field "name" to "nama" in users table:
```
SQLSTATE[HY000]: General error: 1364 Field 'nama' doesn't have a default value (Connection: mysql, SQL: 
insert into `users` (`email`, `password`, `updated_at`, `created_at`) values (admin@email.com, $2y$10$c.WWXlTJ5dr9tJcHi7P2geaSHNxvisXXG7SnrkaODbOX.UuCozxF2, 2023-10-16 19:27:53, 2023-10-16 19:27:53))
```

I want to change the default field name from "name" to "nama" when executing: `php artisan make:filament-user`

**Comment by leandrocfe** [comment]
This command will work if you have a default user table, but you can create a user via [laravel seed](https://laravel.com/docs/10.x/seeding)
You should change in `app/Models/User.php`, factory, .. as well

```php
protected $fillable = [
        'nama',
...
    ];
```

https://filamentphp.com/docs/3.x/panels/users#configuring-the-users-name-attribute

**Reply by siabapet** [reply]
Now i got this error after doing doing your step
<img width="576" alt="error" src="https://github.com/filamentphp/filament/assets/64712237/f7ae6c3a-45f0-47a2-b8e7-4af9d495d73c">

**Comment by siabapet** [comment, marked as answer]
#Solved
It seems that user table doesn't support other language, so the solution is use all field in english
